packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -180,6 +180,15 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     );
   }
 
+  protected okToAddMore(): boolean {
+    const { items } = this.state;
+    const { itemLimit } = this.props;
+    if (typeof itemLimit !== 'undefined' && items.length === itemLimit) {
```

Comment:
minor suggestion; this can be shortened to `itemLimit !== undefined`. AFAIK you only really need to use typeof when you are checking if a global variable like `document` exists (to prevent throwing) or you really care about the primitive type.